cli/docs/cli.json

```
@@ -972,6 +972,96 @@
         {
           "name": "external-ip",
           "subcommands": [
+            {
+              "name": "attach",
+              "about": "Attach an external IP to an instance",
+              "subcommands": [
+                {
+                  "name": "ephemeral",
+                  "about": "Create and attach an ephemeral IP to an instance",
+                  "args": [
+                    {
+                      "long": "instance",
+                      "short": "i",
+                      "help": "Name or ID of the instance"
+                    },
+                    {
+                      "long": "pool",
+                      "help": "The parent IP pool that an IP will be drawn from. If unset, the default pool is selected"
+                    },
+                    {
+                      "long": "project",
+                      "short": "p",
+                      "help": "Name or ID of the project"
+                    }
+                  ]
+                },
+                {
+                  "name": "floating",
+                  "about": "Attach an existing floating IP to an instance",
+                  "args": [
+                    {
+                      "long": "floating-ip",
+                      "short": "f",
+                      "help": "The name of an existing floating IP within the target instance's project"
+                    },
+                    {
+                      "long": "instance",
+                      "short": "i",
+                      "help": "Name or ID of the instance"
+                    },
+                    {
+                      "long": "project",
+                      "short": "p",
+                      "help": "Name or ID of the project"
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "detach",
+              "about": "Detach an external IP from an instance",
+              "subcommands": [
+                {
+                  "name": "ephemeral",
+                  "about": "Detach and deallocate an ephemeral IP from an instance",
```

```suggestion
                  "about": "Detach and deallocate the ephemeral IP from an instance",
```

Is this right? Do I understand correctly that There Can Be Only One?

---

There can indeed only be one. 😄